### DISCUSSION TITLE
fastfetch-linux-old aarch64

### DISCUSSION THREAD

**Discussion by pax0707** [fastfetch-cli/fastfetch#774] [Ideas]
Any change to add this for Raspberry Pies running Debian 11?

**Comment by CarterLi** [comment]
Why not compile it yourself?

**Comment by pax0707** [comment]
Mostly because I never did it.
;- ) =

OK.
With a bit of googlefu it's done.

**Comment by Guiorgy** [comment]
Shameless plug, [here's a shell script](https://gist.github.com/Guiorgy/c3a38ddb9b6f0e6284891388065b971f) I made that runs daily and builds the latest version of fastfetch on my machine, you can try using it if you want.
